src/OpenSpaceToolkit/Core/Error/RuntimeError.cpp

```diff
@@ -1,6 +1,8 @@
 /// Apache License 2.0
 
 #include <OpenSpaceToolkit/Core/Error/RuntimeError.hpp>
+#include <boost/stacktrace.hpp>
+#include <iostream>
```

Comment:
With the reformat, I believe the import ordering as been reversed, right?

```suggestion
#include <iostream>

#include <boost/stacktrace.hpp>

#include <OpenSpaceToolkit/Core/Error/RuntimeError.hpp>
```